examples/simple/BUILD

```diff
@@ -0,0 +1,12 @@
+cc_binary(
+    name = "example_simple",
+    srcs = [
+        "library.cc",
```

Comment:
Please consider putting this into a separate cc_library target.

Reply:
Done.